R/pkg/R/mllib.R

```diff
@@ -694,8 +694,8 @@ setMethod("predict", signature(object = "KMeansModel"),
 #' }
 #' @note spark.mlp since 2.1.0
 setMethod("spark.mlp", signature(data = "SparkDataFrame"),
-          function(data, blockSize = 128, layers = c(3, 5, 2), solver = "l-bfgs", maxIter = 100,
-                   tol = 0.5, stepSize = 1, seed = 1) {
+          function(data, blockSize = 128, layers, solver = "l-bfgs", maxIter = 100,
+                   tol = 1E-6, stepSize = 0.03, seed = -763139545) {
```

Comment:
oh, the default seed use ClassName.hashCode so here it is “org.apache.spark.ml.classification.MultilayerPerceptronClassifier”.hashCode() and it equals -763139545